src/main/java/kitchenpos/products/tobe/domain/Product.java

```diff
@@ -0,0 +1,50 @@
+package kitchenpos.products.tobe.domain;
+
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
+public class Product {
+    @Id
+    @Column(name = "id", columnDefinition = "varbinary(16)")
+    private UUID productId;
```

Comment:
다른 변수와 혼동되지 않는다면 중복되는 문맥의 이름을 피하는 것이 어떨까요?

```suggestion
    private UUID id;
```